janus_core/cli/phonons.py

```diff
@@ -258,6 +258,7 @@ def phonons(
         "write_results": True,
         "write_full": write_full,
         "file_prefix": file_prefix,
+        "enable_progress_bar": True
```

Comment:
```suggestion
        "enable_progress_bar": True,
```
To make ruff happy (best merged with other suggestion)